addons/weaponselect/functions/fnc_selectNextGrenade.sqf

```diff
@@ -31,7 +31,7 @@ _currentGrenade = _currentGrenade select 0;
 private _grenadeType = [GVAR(GrenadesAll), GVAR(GrenadesFrag), GVAR(GrenadesNonFrag)] select _type;
 
 // This is faster than checking magazines
-private _grenades = (throwables _unit) apply {_x select 0} select {_x in _grenadeType};
+private _grenades = (throwables _unit) apply {_x select 0} select {private _i = _x; _grenadeType findIf {_x == _i} != -1};
```

Comment:
`toLowerANSI`+`isEqualTo` would still be faster, so let's optimize their uncaring laziness with that at least:
````suggestion
private _grenades = (throwables _unit) apply {_x select 0} select {private _i = _x; _grenadeType findIf {_x isEqualTo toLowerANSI _i} isNotEqualTo -1};
```

Reply:
Actually, profiling with [ADT](https://steamcommunity.com/sharedfiles/filedetails/?id=2369477168) shows an increase in execution time, `~0.09ms` vs the current `~0.042ms`.